public/components/content-list-drawer/_content-list-drawer.scss

```diff
@@ -72,6 +72,26 @@
         font-weight: bold;
         padding-bottom:10px;
         border-bottom:4px solid $c-grey-300;
+
+        &--closed {
+            writing-mode: vertical-lr;
```

Comment:
For the people that are stuck on older machines because of inCopy, do the installed browser versions on the managed machines support `writing-mode`? Do we have those versions written down anywhere?